package.json

```diff
@@ -19,9 +19,11 @@
   "scripts": {
     "postinstall": "cd lib || builder run npm:postinstall",
```

Comment:
Another note: since this repo does not have an `.npmignore` file, installing from git should currently be possible. The `publishr` package swoops in when you need to omit some files from the npm repo but still want to be installable from git.